GameData/RealismOverhaul/Engine_Configs/XLR11_Config.cfg

```diff
@@ -242,8 +266,61 @@
 				key = 1 208
 			}
 		}
+		CONFIG
+		{
+			name = XLR35-RM-1
+			minThrust = 9.39
+			maxThrust = 27.56
+			massMult = 0.8726
+			heatProduction = 100
+			pressureFed = True
+			%ignitions = 1
+			gimbalRange = 5 // Guess
```

Comment:
If you manually define gimbal range in one config, you must manually define gimbal range in all configs, otherwise bad things™ may occur

Reply:
Will change. Curious though, doe sit cause Bad Things™ even with a global part declaration for gimbal range? That's weird.

Reply:
If you define a value in one CONFIG you have to define it in all of them. Because otherwise when you swap to the config it gets applied...but when you swap away from the config it doesn't get unapplied.

Does the gimbal actually work in your tests? There's no transform specified in your MODULE declaration...

Reply:
It "worked" in that I accidentally tested it on a craft with all-moving fins... forgot to add the transform before, fixed now.